gb-dbg/src/debugger/status_bar.rs

```diff
@@ -22,31 +22,31 @@ impl StatusBar {
             ui.separator();
             ui.horizontal(|ui| {
                 display_flags(ui, u16::from(regs.cpu_get(CpuRegs::AF)));
-                display_key(ui, u16::from(regs.io_get(IORegs::Joy)))
+                display_key(ui, u16::from(regs.io_get(IORegs::Joy)));
             });
         });
     }
 }
 
 fn display_key(ui: &mut Ui, key: u16) {
-    let mut key_display: Vec<char> = vec!['_'; 6];
+    let mut key_display: Vec<&str> = vec!["___ "; 6];
     if key & RIGHT_OR_A == 0 {
-        key_display[0] = 'R';
+        key_display[0] = "➡/A ";
     }
     if key & LEFT_OR_B == 0 {
-        key_display[1] = 'L';
+        key_display[1] = "⬅/B ";
     }
     if key & UP_OR_SELECT == 0 {
-        key_display[2] = 'U';
+        key_display[2] = "⬆/S ";
     }
     if key & DOWN_OR_START == 0 {
-        key_display[3] = 'D';
+        key_display[3] = "⬇/S ";
     }
     if key & SELECT_DIRECTION == 0 {
-        key_display[4] = 'S';
+        key_display[4] = "SLD ";
     }
     if key & SELECT_ACTION == 0 {
-        key_display[5] = 'A';
+        key_display[5] = "SLA ";
```

Comment:
Wouldn't it be more readable to use the `SLD` and `SLA` conditionally to know which exactly which button have been pressed ?

Reply:
it will be but the joypad needs to be fixed first